backend/pkg/kubeconfig/kubeconfig_test.go

```diff
@@ -76,7 +77,7 @@ func TestLoadContextsFromKubeConfigFile(t *testing.T) {
 		contexts, errs := kubeconfig.LoadContextsFromFile(kubeConfigFile, kubeconfig.KubeConfig)
 		require.NotEmpty(t, errs, "Expected some errors for partially valid file")
 		require.NotEmpty(t, contexts, "Expected some valid contexts from partially valid file")
-		require.Equal(t, 1, len(contexts), "Expected one context from the partially valid file")
+		require.Equal(t, 3, len(contexts), "Expected one context from the partially valid file")
```

Comment:
Seems there's 3 now, but the comment says one?

It looks like there's only 1 valid context, and 2 invalid ones in the test file kubeconfig_partialcontextvalid. So the test should check that, and it doesn't at the moment.

Reply:
The scope of the PR has been changed, it now doesn't return partial valid contexts, it returns all contexts. Will edit this, thanks